### DISCUSSION TITLE
Help IGN for Mytoubook

### DISCUSSION THREAD

**Discussion by Yannos60** [mytourbook/mytourbook#956] [Q&A]
Bonjour, 
I'm trying desperately to load IGN SCAN 25 on MyTourBook.
I've got an API key for free on Geoservice site but impossible to load on MTB/Mapsupplier/New WMS
Is there a simple way to do it ? 
Thanks for your help.

Yannos

**Comment by FJBDev** [comment]
Salut @Yannos60 

This is what IGN told me in 2021:
And also what I read [here](https://georezo.net/forum/viewtopic.php?id=122208#p338756)

```
Bonjour,

Depuis le 1er janvier 2021, les ressources SCAN25, Scan 100 et SCAN OACI sont payantes.
Deux options s'offrent à vous :
- Utiliser la ressource PLANIGN
- Une mise en relation avec l'agence régionale la plus proche de chez vous pour établissement d'un devis.
```

However, I have created a file to easily add the IGN maps to MyTourbook.
You can download it [here](https://mytourbook.sourceforge.io/mytourbook/index.php/map-provider)

![image](https://user-images.githubusercontent.com/25235665/198684030-c3262259-6d96-498a-a36c-c2b471248164.png)

I hope that helps!
Frederic

**Comment by Yannos60** [comment]
Bonjour Frédéric

 

Thanks for your answer.

I’ve seen that you have a French first name. I don’t know wether I can speak in French or not!

However for the moment I’m going on in English.

I have downloaded Géoportail Plan IGN v2 which is already very helpful and I thank you very much for this!

 

However what I do not understand is that I have a API key for free that I got on https://geoservices.ign.fr/telechargement.

This API Key allows me to watch my tours with IGN scan 25,   on an old version of Sporttracks 3. 

 

 

 



 

Some years ago, and before ST3 finishes I’ve downloaded a widget that allows to access to this map.

Now , I’m on MyTourBook  but I still keep on watching ST3 to visualize my tour in details.

Should MTB have the same, it should be great!

 

Best regards

 

Yann

 

De : FJBDev ***@***.***> 
Envoyé : vendredi 28 octobre 2022 18:14
À : mytourbook/mytourbook ***@***.***>
Cc : Yannos60 ***@***.***>; Mention ***@***.***>
Objet : Re: [mytourbook/mytourbook] Help IGN for Mytoubook (Discussion #956)

 

Salut @Yannos60 <https://github.com/Yannos60> 

This is what IGN told me in 2021:
And also what I read here <https://georezo.net/forum/viewtopic.php?id=122208#p338756> 

Bonjour,
 
 
 
Depuis le 1er janvier 2021, les ressources SCAN25, Scan 100 et SCAN OACI sont payantes.
 
Deux options s'offrent à vous :
 
- Utiliser la ressource PLANIGN
 
- Une mise en relation avec l'agence régionale la plus proche de chez vous pour établissement d'un devis.
 

However, I have created a file to easily add the IGN maps to MyTourbook.
You can download it here <https://mytourbook.sourceforge.io/mytourbook/index.php/map-provider> 

 <https://user-images.githubusercontent.com/25235665/198684030-c3262259-6d96-498a-a36c-c2b471248164.png> 

I hope that helps!
Frederic

**Reply by FJBDev** [reply]
@Yannos60 

Oui on peut parler francais 😄 
En general, je prefere l'anglais pour que chaque utilisateur puisse lire chaque contenu mais vu que les cartes IGN ne s'appliquent en general qu'aux utilisateurs francais, on peut basculer dans la langue de Moliere.

Effectivement, avec une cle, on peut utiliser les SCAN 25 dans MyTourbook mais c'est un procede qui requiert quand meme chaque utilisateur de creer un compte IGN, donner son adresse, etc.....
Je viens de creer un fichier XML qui permettra aux utilisateurs qui possedent une cle de voir les SCAN IGN 25. Je joins le fichier XML ou il vous faudra entrer votre cle comme dans la video que j'ai faite [ici](https://www.youtube.com/watch?v=ufACEVXf0SI)
Si ces cartes SCAN 25 sont utilisees par beaucoup d'utilisateurs, je ne suis pas contre mettre le fichier avec le reste des cartes sur [le site MyTourbook](https://mytourbook.sourceforge.io/mytourbook/index.php/map-provider) mais pour l'instant, vous etes le seul a avoir exprime de l'interet.

[cartes-ign-2022-10-28.txt](https://github.com/mytourbook/mytourbook/files/9892530/cartes-ign-2022-10-28.txt)

![image](https://user-images.githubusercontent.com/25235665/198706530-d1455b6d-63e3-4edc-b1d2-988caa5d413a.png)

**Reply by FJBDev** [reply]
@Yannos60 
>>Now , I’m on MyTourBook  but I still keep on watching ST3 to visualize my tour in details.

Je suis curieux, les cartes IGN sont-elles la seule raison pour laquelle vous utilisez ST3 ?

**Comment by Yannos60** [comment]
Bonjour
Merci pour vos réponses et le lien de la vidéo. Je vais essayer de mettre tout ça en application.  Je comprends que je dois télécharger les fichiers XML. Je vais regarder ça dans geoportail.....mais bon, je ne suis pas un spécialiste de l’informatique et ce n’est pas à 67 ans que je vais le devenir!🤭😉😉

Pour ce qui concerne votre question, oui les cartes  IGN SCAN sont la seule raison pour laquelle j’utilise encore ST3. Elles me permettent de bien visualiser mon parcours en VTT. Auparavant j’utilisais aussi ST3 pour tracer des parcours et je dois dire qu’à ce jour, je n’ai jamais rien  trouvé de plus facile, même pas Openrunner.

Donc, dès que je trouve le moyen d’avoir les cartes IGN SCAN 25 sur MTB, je n’aurai plus aucune raison d’ aller sur ST3 et je le desinstallerai.

Bon week-end 

Yannos


⁣Télécharger BlueMail pour Android ​

Le 29 oct. 2022, 02:44, à 02:44, FJBDev ***@***.***> a écrit:
***@***.*** 
>>>Now , I’m on MyTourBook  but I still keep on watching ST3 to
>visualize my tour in details.
>
>Je suis curieux, les cartes IGN sont-elles la seule raison pour
>laquelle vous utilisez ST3 ?
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/mytourbook/mytourbook/discussions/956#discussioncomment-3999393
>You are receiving this because you were mentioned.
>
>Message ID:
***@***.***>

**Reply by Yannos60** [reply]
Re bonjour, 
Je ne trouve pas le fichier XML que vous avez créé. J'ai juste un fichier Carte -ign 2022 au format texte....

**Reply by FJBDev** [reply]
Le fichier a une extension `.txt` car GitHub refusait l'extension`.xml`. Vous pouvez renommer l'extension en `.xml` meme si cela n'est pas necessaire pour l'import de la carte

**Comment by Yannos60** [comment]
Hello

I’ve just tried to process your solution but unfortunately it doesn’t work. Is W11 the issue ?

 

De : FJBDev ***@***.***> 
Envoyé : samedi 29 octobre 2022 02:45
À : mytourbook/mytourbook ***@***.***>
Cc : Yannos60 ***@***.***>; Mention ***@***.***>
Objet : Re: [mytourbook/mytourbook] Help IGN for Mytoubook (Discussion #956)

 

@Yannos60 <https://github.com/Yannos60> 

Now , I’m on MyTourBook but I still keep on watching ST3 to visualize my tour in details.

Je suis curieux, les cartes IGN sont-elles la seule raison pour laquelle vous utilisez ST3 ?

**Reply by FJBDev** [reply]
Pouvez vous attacher un log derreur ou une capture decran afin de pouvoir visualiser ce qui ne marche pas?

**Comment by Yannos60** [comment]
Merci encore de votre aide.

Voici le message d’erreur qui apparaît.

 



 

 

 

Yannos

De : FJBDev ***@***.***> 
Envoyé : samedi 29 octobre 2022 16:42
À : mytourbook/mytourbook ***@***.***>
Cc : Yannos60 ***@***.***>; Mention ***@***.***>
Objet : Re: [mytourbook/mytourbook] Help IGN for Mytoubook (Discussion #956)

 

Pouvez vous attacher un log derreur ou une capture decran afin de pouvoir visualiser ce qui ne marche pas?

**Reply by FJBDev** [reply]
> Merci encore de votre aide. Voici le message d’erreur qui apparaît. Yannos

Desole mais je ne vois aucun message d'erreur dans votre message

**Comment by Yannos60** [comment]
J’ai bien renommé le fichier en XML 

 



 

De : FJBDev ***@***.***> 
Envoyé : samedi 29 octobre 2022 17:09
À : mytourbook/mytourbook ***@***.***>
Cc : Yannos60 ***@***.***>; Mention ***@***.***>
Objet : Re: [mytourbook/mytourbook] Help IGN for Mytoubook (Discussion #956)

 

Le fichier a une extension .txt car GitHub refusait l'extension.xml. Vous pouvez renommer l'extension en .xml meme si cela n'est pas necessaire pour l'import de la carte

**Comment by Yannos60** [comment]
Oui mais quand je suis votre explicatif et qu’ensuite , je clique sur « EDIT » il ne se passe rien. J’ai comme l’impression que le fichier est bloqué….

 

De : FJBDev ***@***.***> 
Envoyé : samedi 29 octobre 2022 17:19
À : mytourbook/mytourbook ***@***.***>
Cc : Yannos60 ***@***.***>; Mention ***@***.***>
Objet : Re: [mytourbook/mytourbook] Help IGN for Mytoubook (Discussion #956)

 

Merci encore de votre aide. Voici le message d’erreur qui apparaît. Yannos

Desole mais je ne vois aucun message d'erreur dans votre message

**Reply by FJBDev** [reply]
> Oui mais quand je suis votre explicatif et qu’ensuite , je clique sur « EDIT » il ne se passe rien. J’ai comme l’impression que le fichier est bloqué…. 

Ok, mais sans message d'erreur ou captiure d'ecran, ca va etre dur de diagnostiquer tout ca.

Sportivement
Frederic

**Comment by Yannos60** [comment]
La capture d’écran n’est elle pas suffisante ?

 

 

 

De : FJBDev ***@***.***> 
Envoyé : samedi 29 octobre 2022 17:35
À : mytourbook/mytourbook ***@***.***>
Cc : Yannos60 ***@***.***>; Mention ***@***.***>
Objet : Re: [mytourbook/mytourbook] Help IGN for Mytoubook (Discussion #956)

 

Oui mais quand je suis votre explicatif et qu’ensuite , je clique sur « EDIT » il ne se passe rien. J’ai comme l’impression que le fichier est bloqué….

Ok, mais sans message d'erreur ou captiure d'ecran, ca va etre dur de diagnostiquer tout ca.

Sportivement
Frederic

**Comment by Yannos60** [comment]
La fenêtre indique cependant, «  Il y a des erreurs dans le fichier….. contenu non autorisé dans le prologue » c/user…

Je n’y comprends rien du tout ☹ ☹

 

De : FJBDev ***@***.***> 
Envoyé : samedi 29 octobre 2022 17:35
À : mytourbook/mytourbook ***@***.***>
Cc : Yannos60 ***@***.***>; Mention ***@***.***>
Objet : Re: [mytourbook/mytourbook] Help IGN for Mytoubook (Discussion #956)

 

Oui mais quand je suis votre explicatif et qu’ensuite , je clique sur « EDIT » il ne se passe rien. J’ai comme l’impression que le fichier est bloqué….

Ok, mais sans message d'erreur ou captiure d'ecran, ca va etre dur de diagnostiquer tout ca.

Sportivement
Frederic

**Reply by FJBDev** [reply]
Desole mais je ne vois aucune capture decran dans vos messages :-(

**Comment by Yannos60** [comment]
Zut… Ils sont bloqués alors. Je vous les envoyer en zip….

 

 

 

De : FJBDev ***@***.***> 
Envoyé : samedi 29 octobre 2022 17:51
À : mytourbook/mytourbook ***@***.***>
Cc : Yannos60 ***@***.***>; Mention ***@***.***>
Objet : Re: [mytourbook/mytourbook] Help IGN for Mytoubook (Discussion #956)

 

Desole mais je ne vois aucune capture decran dans vos messages :-(

**Reply by Yannos60** [reply]
Voici les captures 
![1](https://user-images.githubusercontent.com/116881134/198841603-8c84ae14-b67e-4e79-9c2d-3d5a44a603fa.jpg)
![2](https://user-images.githubusercontent.com/116881134/198841606-e21f37a9-1c68-49bc-b0d0-58e3ad6e862b.jpg)

**Reply by FJBDev** [reply]
Ok, si vous pouvez aussi attacher le fichier .log, ca aideraot aussi a diagnostiquer lerreur

Merci

**Reply by FJBDev** [reply]
Erreur bizarre, le log en dira plus...

**Comment by Yannos60** [comment]
Heu…. Je trouve ce fichier où ?

 

 

De : FJBDev ***@***.***> 
Envoyé : samedi 29 octobre 2022 18:08
À : mytourbook/mytourbook ***@***.***>
Cc : Yannos60 ***@***.***>; Mention ***@***.***>
Objet : Re: [mytourbook/mytourbook] Help IGN for Mytoubook (Discussion #956)

 

Ok, si vous pouvez aussi attacher le fichier .log, ca aideraot aussi a diagnostiquer lerreur

Merci

**Reply by Yannos60** [reply]
Ça y est, j'ai trouvé !
![3](https://user-images.githubusercontent.com/116881134/198842292-9b6018a8-0fb6-4c6d-90a6-985c72efebae.jpg)

**Reply by FJBDev** [reply]
Super, si vous pouviez m'envoyer le contenu de l'erreur ou alors le fichier complet qui se trouve ici `C:\Users\NOMDUTILISATEUR\mytourbook\.metadata`

Merci

**Reply by Yannos60** [reply]
Aie, Le fichier est interdit : 
" This file is hidden. Try with another file"

**Reply by FJBDev** [reply]
Decidement.....

Pouvez vous menvoyer alors le texte qui saffiche lorsque vous double cliquez sur lentree du log correspondant a lerreur?

**Reply by FJBDev** [reply]
Pouvez-vous essayer de changer la clef api directement dans le fichier ? Il vous suffira de remplacer CLEFAPI par votre clef aux deux endroits

**Reply by Yannos60** [reply]
Voilà, J'espère que cela vous  parlera plus qu'à moi;;;

![4](https://user-images.githubusercontent.com/116881134/198845295-01e14eb7-11f9-4f27-be60-2bbd2da2d21e.jpg)

**Reply by Yannos60** [reply]
Ok mais quel fichier ?

**Reply by FJBDev** [reply]
Cette entree de log n'est pas la bonne. Il me faudrait l'entree correspondant a l'erreur "Contenu non autorise", qui a aussi une croix rouge.
L'entree que vous avez mise est une un message d'information

**Reply by Yannos60** [reply]
Voilà
![5](https://user-images.githubusercontent.com/116881134/198845659-26112fba-09f6-4e8c-a3a9-6094c7b544ba.jpg)

**Reply by Yannos60** [reply]
SUPER !!!!!!!!!!!!!!
Je viens de remplacer la CLEFAPI aux 2 endroits,  comme vous me l'avez indiqué et ÇA MARCHE!
Merci infiniment pour votre aide. Je suis ravi!.

Bonne soirée.

Yannos

**Reply by FJBDev** [reply]
Super, merci!
Bizarre, cela veut dire qu'apparemment le fichier XML serait non conforme.....

Pouvez-vous ouvrir le fichier XML avec un editeur de texte par exemple et attacher la capture d'ecran ici ? Peut etre que le telechargement s'est mal fait et que le fichier XML est incomplet ?

**Reply by FJBDev** [reply]
> SUPER !!!!!!!!!!!!!! Je viens de remplacer la CLEFAPI aux 2 endroits, comme vous me l'avez indiqué et ÇA MARCHE! Merci infiniment pour votre aide. Je suis ravi!.
> 
> Bonne soirée.
> 
> Yannos

Super, je suis content que ca marche 😄 

Sportivement,
Frederic

**Comment by Yannos60** [comment]
Je viens juste de quitter mon bureau. Je  ferai ça demain  sans faute..
Ce que je peux vous dire c´est que j´ai fait la  modification " CLEFAPI" dans l´éditeur txt car l´éditeur XML ne présentait pas cet item.

Merci et bonne soirée 

Yann 

⁣Télécharger BlueMail pour Android ​

Le 29 oct. 2022, 19:47, à 19:47, FJBDev ***@***.***> a écrit:
>Super, merci!
>Bizarre, cela veut dire qu'apparemment le fichier XML serait non
>conforme.....
>
>Pouvez-vous ouvrir le fichier XML avec un editeur de texte par exemple
>et attacher la capture d'ecran ici ? Peut etre que le telechargement
>s'est mal fait et que le fichier XML est incomplet ?
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/mytourbook/mytourbook/discussions/956#discussioncomment-4007212
>You are receiving this because you were mentioned.
>
>Message ID:
***@***.***>

**Comment by Yannos60** [comment]
Bonjour, 
It seems that IGN has changed it's platform. I Cannot still use SCAN 25 that allows me to see more acurately may various tours.
Have you any idea how I could charge it again in 2D maps ?
Thanks a lot in advance.
Yannos

**Reply by FJBDev** [reply]
Hello,

From my understanding, the SCAN25 is not available as of today.
Even though `GeoServices` advertises [here ](https://geoservices.ign.fr/services-geoplateforme-diffusion)that it's available as below:
```
GEOGRAPHICALGRIDSYSTEMS.MAPS<!-- Cartes IGN -->
GEOGRAPHICALGRIDSYSTEMS.MAPS.SCAN25TOUR<!-- Cartes SCAN 25® Touristique -->
```
However, when I look at their [GetCapabilities ](https://data.geopf.fr/wmts?SERVICE=WMTS&VERSION=1.0.0&REQUEST=GetCapabilities)response, I don't see either of the layers mentioned above
Additionally, if I try to request a tile manually in the browser, I get the below error for that tile:
`https://data.geopf.fr/wmts?SERVICE=WMTS&REQUEST=GetTile&VERSION=1.0.0&LAYER=GEOGRAPHICALGRIDSYSTEMS.MAPS&STYLE=normal&FORMAT=image/png&TILEMATRIXSET=PM&TILEMATRIX=11&TILEROW=731&TILECOL=1063`

```
<ExceptionReport>
<Exception exceptionCode="InvalidParameterValue"> Layer GEOGRAPHICALGRIDSYSTEMS.MAPS inconnu. </Exception>
</ExceptionReport>
```

Therefore, instead, I use the `PLANIGNV2` in MyTourbook and it doesn't look too bad, see my screensot below

![image](https://github.com/user-attachments/assets/1ab6a41c-5017-4896-8aef-a84eee1c8750)

**Comment by Yannos60** [comment]
Merci FJB. 
That's what I have understood. I use, of course The Planning V2 but it's not so acurante as SCAN 25. We have already spoken on this issue (Cf messages above !) .I hope it' will be soon available again.

Best regards.

Yannos

**Reply by FJBDev** [reply]
Yannos,

De ce que je comprends, les cartes IGN 25 sont des donnees "non-libres" qui requierent un access specifique
[Source](https://geoservices.ign.fr/actualites/2023-11-20-acces-donnesnonlibres-gpf)

`Pour y accéder (dans la limite des droits accordés à l'utilisateur), il est nécessaire de disposer d’une clé à déclarer en paramètre ou en en-tête des appels au point d’accès privé.`

**Comment by Yannos60** [comment]
FJB, Oui j’ai vu cela. Je crois cependant avoir lu que l’accès était gratuit pour les particuliers mais avec un nombre limité de chargements, ce qui me convient parfaitement.  Par contre,  je n’ai pas vu où je devais faire la demande . De toutes façons, comme tu l'as indiqué les couches ad hoc ne sont pas livrables, donc je patiente en espérant que planning V2 sera maintenu.
 C’est bien compliqué tout ça. 
Si j’ai du neuf sur le sujet, je me permettrai de te recontacter. Et de ton côté, je serai vraiment content si tu trouve qq chose.
Bien à toi.

Yannos

**Reply by FJBDev** [reply]
Oui, je trouve ca bien complique aussi.
S'il est vraiment possible pour chaque particulier d'avoir acces a cette couche, je suis partant pour refaire un tutoriel pour integrer cette couche dans MT mais comme toi je n'ai pas vu non plus ou et comment faire la demande.

Cordialement

**Comment by travelerdg** [comment]
Depuis le 30 septembre les requêtes en wxs.ign.fr ne fonctionnent plus et ont été basculées en data.geopf.fr. Une modification des URL pour les cartes de l'IGN est donc nécessaire. Le site https://geoservices.ign.fr/bascule-vers-la-geoplateforme#changement donne toutes les infos concernant les couches disponibles  pour les différents services de ressources (Voir le fichier csv). Pour le Scan 25 Touristique en WMTS et WMS Raster, l'accès est non libre mais une clé générique temporaire est donnée par l'IGN (apikey=ign_scan_ws) et fonctionne pour chaque particulier. Je l'ai testé sur MT avec l'outil de personnalisation des fournisseurs de cartes. Le fonds de carte est OK.

Description de l'url utilisé :  https://data.geopf.fr/private/wmts?apikey=ign_scan_ws&LAYER=GEOGRAPHICALGRIDSYSTEMS.MAPS.SCAN25TOUR&EXCEPTIONS=text/xml&FORMAT=image/jpeg&SERVICE=WMTS&VERSION=1.0.0&REQUEST=GetTile&STYLE=normal&TILEMATRIXSET=PM&TILEMATRIX={zoom}&TileCol={x}&TileRow={y}

Cordialement.

**Comment by Yannos60** [comment]
Bonjour, 

 

 

Merci pour cette bonne nouvelle !

Cependant j’ai copié l’adresse que vous avez indiquée mais , malheureusement , il ne se passe rien… 

Je vous joins la copie d’écran. 

 

Je ne sais pas quelle erreur j’ai commis !

 

Bien à vous.

 

Yann

De : travelerdg ***@***.***> 
Envoyé : mercredi 9 octobre 2024 11:12
À : mytourbook/mytourbook ***@***.***>
Cc : Yannos60 ***@***.***>; Mention ***@***.***>
Objet : Re: [mytourbook/mytourbook] Help IGN for Mytoubook (Discussion #956)

 

Depuis le 30 septembre les requêtes en wxs.ign.fr ne fonctionnent plus et ont été basculées en data.geopf.fr. Une modification des URL pour les cartes de l'IGN est donc nécessaire. Le site [( https://geoservices.ign.fr/bascule-vers-la-geoplateforme#changement) ] donne toutes les infos concernant les couches disponibles pour les différents services de ressources (Voir le fichier csv). Pour le Scan 25 Touristique en WMTS et WMS Raster, l'accès est non libre mais une clé générique temporaire est donnée par l'IGN (apikey=ign_scan_ws) et fonctionne pour chaque particulier. Je l'ai testé sur MT avec l'outil de personnalisation des fournisseurs de cartes. Le fonds de carte est OK.

Description de l'url utilisé : https://data.geopf.fr/private/wmts?apikey=ign_scan_ws <https://data.geopf.fr/private/wmts?apikey=ign_scan_ws&LAYER=GEOGRAPHICALGRIDSYSTEMS.MAPS.SCAN25TOUR&EXCEPTIONS=text/xml&FORMAT=image/jpeg&SERVICE=WMTS&VERSION=1.0.0&REQUEST=GetTile&STYLE=normal&TILEMATRIXSET=PM&TILEMATRIX=%7Bzoom%7D&TileCol=%7Bx%7D&TileRow=%7By%7D> &LAYER=GEOGRAPHICALGRIDSYSTEMS.MAPS.SCAN25TOUR&EXCEPTIONS=text/xml&FORMAT=image/jpeg&SERVICE=WMTS&VERSION=1.0.0&REQUEST=GetTile&STYLE=normal&TILEMATRIXSET=PM&TILEMATRIX={zoom}&TileCol={x}&TileRow={y}

Cordialement.

**Comment by travelerdg** [comment]
Il s'agit d'une DESCRIPTION de l'url utilisable pour MT dans la rubrique Outils/Préférences/Cartes 2D/Fournisseurs de cartes. 
C'est normal qu'il ne se passe rien. Il faudrait donner des valeurs aux paramètres de zoom et de coordonnées (X,Y) de la tuile .

Vous pouvez tester cet url  avec un zoom 11 et les coordonnées X= 1063 et Y = 711 : 

https://data.geopf.fr/private/wmts?apikey=ign_scan_ws&LAYER=GEOGRAPHICALGRIDSYSTEMS.MAPS.SCAN25TOUR&EXCEPTIONS=text/xml&FORMAT=image/jpeg&SERVICE=WMTS&VERSION=1.0.0&REQUEST=GetTile&STYLE=normal&TILEMATRIXSET=PM&TILEMATRIX=11&TileCol=1063&TileRow=711

et vérifier qu'il retourne bien une tuile au format jpeg de la couche SCAN 25 Touristique.

**Comment by Yannos60** [comment]
Merci  de votre réponse.

J’ai chargé l’URL avec les valeurs que vous m’avez communiquées (Je ne sais d’ailleurs pas à quoi correspondent ces valeurs) et j’ai bien une tuile IGN qui s’affiche. 

Le problème est que je n’ai qu’une tuile ! Voir si joint la capture d’un trajet 

Je suis perdu !

 

 

De : travelerdg ***@***.***> 
Envoyé : mercredi 9 octobre 2024 15:08
À : mytourbook/mytourbook ***@***.***>
Cc : Yannos60 ***@***.***>; Mention ***@***.***>
Objet : Re: [mytourbook/mytourbook] Help IGN for Mytoubook (Discussion #956)

 

Il s'agit d'une DESCRIPTION de l'url utilisable pour MT dans la rubrique Outils/Préférences/Cartes 2D/Fournisseurs de cartes.
C'est normal qu'il ne se passe rien. Il faudrait donner des valeurs aux paramètres de zoom et de coordonnées (X,Y) de la tuile .

Vous pouvez tester cet url avec un zoom 11 et les coordonnées X= 1063 et Y = 711 :

https://data.geopf.fr/private/wmts?apikey=ign_scan_ws <https://data.geopf.fr/private/wmts?apikey=ign_scan_ws&LAYER=GEOGRAPHICALGRIDSYSTEMS.MAPS.SCAN25TOUR&EXCEPTIONS=text/xml&FORMAT=image/jpeg&SERVICE=WMTS&VERSION=1.0.0&REQUEST=GetTile&STYLE=normal&TILEMATRIXSET=PM&TILEMATRIX=11&TileCol=1063&TileRow=711> &LAYER=GEOGRAPHICALGRIDSYSTEMS.MAPS.SCAN25TOUR&EXCEPTIONS=text/xml&FORMAT=image/jpeg&SERVICE=WMTS&VERSION=1.0.0&REQUEST=GetTile&STYLE=normal&TILEMATRIXSET=PM&TILEMATRIX=11&TileCol=1063&TileRow=711

et vérifier qu'il retourne bien une tuile au format jpeg de la couche SCAN 25 Touristique.

**Comment by travelerdg** [comment]
C'est simplement un test pour vérifier que l'adresse de l'url est correcte et que la requête est bien prise en compte. J'avais envoyé ce message pour que Frédéric fasse les modifications d' URL pour avoir de nouveau les cartes IGN (Satellite, Plan IGN V2,  IGN, ...) dans MT. 
Avec toutes ces infos, il sera tout à fait capable d' ajouter aussi le SCAN 25 Touristique. 

Attendons sa réponse.

**Reply by FJBDev** [reply]
> C'est simplement un test pour vérifier que l'adresse de l'url est correcte et que la requête est bien prise en compte. J'avais envoyé ce message pour que Frédéric fasse les modifications d' URL pour avoir de nouveau les cartes IGN (Satellite, Plan IGN V2, IGN, ...) dans MT. Avec toutes ces infos, il sera tout à fait capable d' ajouter aussi le SCAN 25 Touristique.
> 
> Attendons sa réponse.

@travelerdg Merci d'avoir trouve l'URL pour faire marcher les SAN IGNs dans MyTourbook.

Ci-joint est le fichier XML qui peut etre importe dans MT pour afficher ces couches (L'extension `txt `doit etre renommee en `xml `car GitHub refuse ce type d'extension)
Ca marche tres bien chez moi ! 😄 

Une question me brule les levres: La cle api `ign_scan_ws `est-elle legale d'utilisation pour un public comme les utilisateurs de MT ?

![image](https://github.com/user-attachments/assets/1e9add3d-9665-48ae-91bf-d9f4b558da97)
[geoportail-ign-scans-2024-10-09.txt](https://github.com/user-attachments/files/17319804/geoportail-ign-scans-2024-10-09.txt)

**Reply by travelerdg** [reply]
C'est assez ambigu. L'accès est non libre mais l'IGN fournit cette clé api partagée qui fonctionne pour tous les utilisateurs (y compris ceux de MT). D'après les infos données sur cette page web : https://geoservices.ign.fr/actualites/2023-11-20-acces-donnesnonlibres-gpf, il s'agit de conditions d'utilisation entrainant un usage gratuit donc cela semble tout à fait légal. 

Une dernière demande : Pourrais tu mettre à jour sur le site de MT les fichiers Geoportail Plan IGN V2.xml et Geoportail Photographies aériennes HR.xml qui sont devenus obsolètes ?

Merci d'avance

**Reply by FJBDev** [reply]
> Une dernière demande : Pourrais tu mettre à jour sur le site de MT les fichiers Geoportail Plan IGN V2.xml et Geoportail Photographies aériennes HR.xml qui sont devenus obsolètes ?

@travelerdg I don't have the permission to update the MT's website but I have attached the latest file for IGN and maybe @wolfgang-ch can update them ?

[prof-geoportail-maps-2024-10-16.txt](https://github.com/user-attachments/files/17481476/prof-geoportail-maps-2024-10-16.txt)

**Reply by wolfgang-ch** [reply]
> @travelerdg I don't have the permission to update the MT's website but I have attached the latest file for IGN and maybe @wolfgang-ch can update them ?

It will be available when MT 24.11 is published because I've already prepared the website for the new version

![image](https://github.com/user-attachments/assets/680c16e5-90f0-4a56-bc65-18ff85d7551b)

**Comment by Yannos60** [comment]
Ah OK. Merci infiniment. 
Yann

⁣Télécharger BlueMail pour Android ​

Le 9 oct. 2024, 16:58, à 16:58, travelerdg ***@***.***> a écrit:
